test/plugins/tokens.test.js

```diff
@@ -436,4 +440,19 @@ describe('token plugin test', () => {
             });
         });
     });
+
+    describe('Logging suppresses API tokens', () => {
+        it('does not print API tokens in logs', () => {
+            const source = new PassThrough({ objectMode: true });
+            const result = new PassThrough({ objectMode: true });
+
+            source.write(`This is a string with a token in it! ${testTokenWithValue.value}`);
+
+            source.pipe(suppressAPITokens).pipe(result);
+
+            result.on('data', (chunk) => {
```

Comment:
Good call! That was exactly what was happening, I had a mistake in my regex that I didn't even notice.